Formula/c/cryptol.rb

```diff
@@ -21,7 +19,7 @@ class Cryptol < Formula
   end
 
   depends_on "cabal-install" => :build
-  depends_on "ghc@9.4" => :build
```

Comment:
does not support ghc9.8
https://github.com/GaloisInc/cryptol/blob/master/.github/workflows/ci.yml#L68